docs/python/examples.ipynb

```diff
@@ -471,7 +471,7 @@
    "metadata": {},
    "outputs": [],
    "source": [
-    "model = ribasim.Model.from_toml(datadir / \"basic/ribasim.toml\")"
+    "model = ribasim.Model(filepath=datadir / \"basic/ribasim.toml\")"
```

Comment:
I liked having separate constructors here. Is there anything blocking us from being explicit here?
Also, what is the behavior if I now specify `filepath` as well as other parameters? Will the TOML or the manually set parameters take precedence?

Reply:
Hmm, I found it confusing. Would `read` also be acceptable (as there's a `write`)?

Manually set should override, I've fixed that now.

Reply:
`read` sounds good. There is no alternative to our toml.